Automattic-Tracks-iOS/LogFile.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+public class LogFile {
```

Comment:
Will the complete implementation require this type to be `public`? Will the log processing be in the main app code?

Reply:
Yep – it's used in delegate protocols to provide event data to the host app (or test target), so it makes sense for this to be public :)

Reply:
Thanks for clarifying!